addons/pkg-maven/common/src/main/java/org/commonjava/indy/pkg/maven/content/MavenMetadataGenerator.java

```diff
@@ -597,12 +597,9 @@ private Metadata generateGroupMetadata( final Group group, final List<ArtifactSt
         if ( versions != null && !versions.isEmpty() )
         {
             merger.sortVersions( master );
-
-            return master;
         }
 
-        logger.error( "The group metadata generation is not successful for path: {} in group: {}", path, group );
-        return null;
+        return master;
```

Comment:
Or here maybe we should add a master.getPlugins!=null && !master.getPlugins.isEmpty but not directly return this master due to the metadata specification?

Reply:
Exactly! I just updated it.